addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -310,6 +316,9 @@
                             <a t-attf-href="/my/opportunity/#{opp.id}"><span t-field="opp.name"/></a>
                         </td>
                         <td><span t-field="opp.contact_name" /></td>
+                        <td><span t-field="opp.email_from" /></td>
```

Comment:
You can remove the space before the closing tag.

So:
`<td><span t-field="opp.email_from"/></td>`

Instead of:
`<td><span t-field="opp.email_from" /></td>`

You can also remove that space from the "contact_name" while we are modifying this.

(Same for the lines above).